lib/image_scraper/client.rb

```diff
@@ -12,7 +12,9 @@ class Client
     attr_reader :uri, :error
 
     def initialize(url, options = {})
-      options.reverse_merge!(convert_to_absolute_url: true, include_css_images: true, include_css_data_images: false)
+      defaults = { convert_to_absolute_url: true, include_css_images: true, include_css_data_images: false }
+      options.merge!(defaults)
```

Comment:
this is doing the same thing as `reverse_merge!`, but doesn't rely on ActiveSupport